layers/merton.map

```diff
@@ -31,4 +31,57 @@ MAP
     END
   END
 
+  LAYER
+    NAME "all_parks"
+    TYPE POLYGON
+    STATUS ON
+    CONNECTIONTYPE UNION
+    METADATA
+      "wfs_title"         "All Parks" ##REQUIRED
+      "wfs_srs"           "EPSG:3857 EPSG:27700" ## REQUIRED
+      "gml_include_items" "Park_Name,Park_ID,UPRN" ## Optional (serves all attributes for layer)
+      "gml_featureid"     "Park_ID" ## REQUIRED
+      "wfs_enable_request" "*"
```

Comment:
Do we want to add "wfs_getfeature_formatlist" "geojson" here and in other layers